Adding TCGA SNV calling to methods (#88)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

### Somatic Mutation Calling
#### SNV and INDEL calling

> For PBTA samples, we used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and VarDict. WXS samples from TCGA were run using Strelka2, Mutect2 and Lancet. TCGA samples were  captured using different WXS target capture kits and all the BED files were downloaded from [`GDC portal`](https://api.gdc.cancer.gov/files).

Reviewer: ```suggestion
For PBTA samples, we used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet  [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
WXS samples from TCGA were run using Strelka2, Mutect2 and Lancet.
TCGA samples were captured using different WXS target capture kits and all the BED files were downloaded from [`GDC portal`](https://api.gdc.cancer.gov/files).
```

> The input interval BED files for both panel and WXS data for PBTA  samples were provided by the manufacturers. All panel and WXS  BED files were padded by 100 bp on each side during Strelka2, Mutect2, and VarDict runs and 400 bp for the Lancet run.  

Reviewer: ```suggestion
The input interval BED files for both panel and WXS data for PBTA  samples were provided by the manufacturers.
For both PBTA and TCGA, all panel and WXS  BED files were padded by 100 bp on each side during Strelka2, Mutect2, and VarDict runs and 400 bp for the Lancet run.  
```

Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained.
The 0.05 VAF increased the true positive rate for INDELs and decreased the false positive rate for SNVs when using VarDict in consensus calling.
The final VCF was filtered for PASS variants with TYPE=StronglySomatic.

> Lancet [@doi:10.1038/s42003-018-0023-9] v1.0.7 [@url:https://github.com/nygenome/lancet] was run using default parameters, unless noted below. For input intervals to Lancet WGS, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from Strelka2 and Mutect2 [@doi:10.1101/623702]. These intervals were then padded by 300 bp on each side during Lancet variant calling. 

Reviewer: ```suggestion
Lancet v1.0.7 was run using default parameters, except for those noted below.
For input intervals to Lancet WGS, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from Strelka2 and Mutect2 [@doi:10.1101/623702].
These intervals were then padded by 300 bp on each side during Lancet variant calling. 
```

These intervals were then padded by 300 bp on each side during Lancet variant calling.
Per recommendations by the New York Genome Center [@doi:10.1101/623702], for WGS samples, the Lancet input intervals described above were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.

##### TCGA data

> We used three variant callers to call SNVs and INDELS in TCGA data from WXS data: Strelka2, Mutect2  and Lancet. Not  all BAM files within the same disease group were  captured  with the same WXS kit. We retreived individual target  capture kit BED files for each BAM file from 

Reviewer: Cite publications for callers?

[`GDC portal`](https://api.gdc.cancer.gov/files). The input  BED files were  padded by 100 bp on each side during Strelka2 and Mutect2 variant calling and 400bp for Lancet variant calling to accomodate local reassembly.
The final Strelka2 VCF was filtered for PASS variants.

> Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].

Reviewer: This appears to be a direct copy of the section above. What about not breaking out the two sections and then just having one paragraph describing differences, if any, between the two workflows?

Author: Hi @cgreene Thank you for your comments! I updated the section where I combined both TCGA and PBTA under SNV calling.

I also wanted to add how many samples we chose from TCGA under each disease group and primary diagnosis. Do you have a recommendation on where that would be most appropriate to add? Is `Data generation` the right spot? 
https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#data-generation
